src/test/ui/lint/must_not_suspend/unit.stderr

```diff
@@ -0,0 +1,23 @@
+error: `Umm` held across a yield point, but should not be
+  --> $DIR/unit.rs:20:9
+   |
+LL |     let _guard = bar();
+   |         ^^^^^^
+LL |     other().await;
+   |     ------------- The value is held across this yield point
+   |
+note: the lint level is defined here
+  --> $DIR/unit.rs:3:9
+   |
+LL | #![deny(must_not_suspend)]
+   |         ^^^^^^^^^^^^^^^^
+   = note: You gotta use Umm's, ya know?
```

Comment:
How were you expecting this would look? It seems fine to me :) but you mentioned in the PR you wanted to improve it.

Reply:
I think it looks hidden under the lint level diagnostic

Reply:
We could maybe change the logic for lints to always include the `the lint level is defined here` last, but that would go beyond the scope of this PR, IMO.

Reply:
Filed #88908 as a followup